app/models/ems_refresh/refreshers/ems_refresher_mixin.rb

```diff
@@ -135,13 +135,13 @@ def parse_targeted_inventory(ems, target, collector)
           _log.debug "#{log_header} Parsing inventory..."
           inventory_collections, = Benchmark.realtime_block(:parse_inventory) do
             provider_module = ManageIQ::Providers::Inflector.provider_module(ems.class).name
-            inventory_target_class = "#{provider_module}::Inventory::Target::#{target.class.name.demodulize}".safe_constantize
-            inventory_target = inventory_target_class.new(target)
+            persister_class = "#{provider_module}::Inventory::Persister::#{target.class.name.demodulize}".safe_constantize
```

Comment:
Are you using strings/`safe_constantize` to avoid class not found?

(basically asking will this work if you don't `.name`, but just reference the class)?